api/proto/teleport/decision/v1alpha1/tls_identity.proto

```diff
@@ -159,6 +160,8 @@ message TLSIdentity {
   // ScopePin is an optional pin that ties the certificate to a specific scope and set of scoped roles. When
   // set, the Roles field must not be set.
   teleport.scopes.v1.Pin scope_pin = 37;
+
+  repeated types.ResourceAccessID allowed_resource_access_ids = 38;
```

Comment:
missing go docs 

How this field differ from  `repeated ResourceId allowed_resource_ids = 31;` ? 

https://github.com/gravitational/teleport/pull/61168/changes#diff-895441f0aa43e71b80a90630fa8076cf8995aeaa7f91fa58c68c4a211eb08191R140